Box.V2/Models/BoxUserFolderCollaborationEventSource.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Newtonsoft.Json;
+
+namespace Box.V2.Models
+{
+    /// <summary>
+    /// There is an inconsistency in the events API where file sources have slightly different field names.
+    /// </summary>
+    public class BoxUserFolderCollaborationEventSource : BoxEntity
+    {
+        public const string FieldFolderId = "folder_id";
+        public const string FieldFolderName = "folder_name";
+        public const string FieldUserId = "user_id";
+        public const string FieldUserName = "user_name";
+        public const string FieldParent = "parent";
+
+        /// <summary>
+        /// The Id of the folder.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldFolderId)]
+        new public string FolderId { get; private set; }
```

Comment:
You shouldn't need `new` on any of these properties, since they doesn't collide with any properties in the base class.